content/verification/va-letter-generator/release-notes/2025-06-09.md

```diff
@@ -0,0 +1,3 @@
+ - The VBA letterhead and logo has been restored to all letters except the Minimum Essential Coverage and Medicare Park D (which keep the VHA letterhead and logo). 
+ - We fixed a defect that caused a benefit award amount to mistakenly display $0 instead of the actual amount. 
+ - We corrected the Veteran Benefit Summary letter to include a 0% disability rating as a service-connected disability (example: service-connected disabilities = Y).
```

Comment:
```suggestion
 - We corrected the Veteran Benefit Summary letter to include a 0% disability rating as a service-connected disability (for example: service-connected disabilities = Y).
```
I'm not totally understanding the example as it relates to the 0% disability rating. Should it = 0% or Y? @lglover-va